cursorless-talon-dev/src/cursorless_dev.py

```diff
@@ -28,3 +28,7 @@ def cursorless_record_that_mark_test():
         actions.user.run_rpc_command(
             "cursorless.recordTestCase", {"captureFinalThatMark": True}
         )
+
+    def cursorless_record_silent():
```

Comment:
i would follow convention in this file and name this `cursorless_record_silent_test`

Reply:
done, thanks